src/frontend/src/components/public/PIAFormTabs/intake/index.tsx

```diff
@@ -67,30 +82,43 @@ export const PIAFormIntake = () => {
             <div className="row">
               <InputText
                 label="Title"
+                fieldMandatoryLabel="(required)"
                 value={intakeForm?.title}
                 onChange={(e) => stateChangeHandler(e.target.value, 'title')}
                 required={true}
               />
+              {validationMessage.piaTitle && (
+                <p className="text-danger"> {validationMessage.piaTitle}</p>
+              )}
             </div>
             <div className="row">
-              <Dropdown
-                id="ministry-select"
-                value={intakeForm?.ministry || ''}
-                label="Ministry"
-                optionalClass="col-md-6"
-                options={MinistryList}
-                changeHandler={(e) =>
-                  stateChangeHandler(e.target.value, 'ministry')
-                }
-                required={true}
-              />
+              <div className="col col-md-6">
+                <Dropdown
+                  id="ministry-select"
+                  fieldMandatoryLabel="(required)"
+                  value={intakeForm?.ministry || ''}
+                  label="Ministry"
+                  options={MinistryList}
+                  changeHandler={(e) =>
+                    stateChangeHandler(e.target.value, 'ministry')
+                  }
+                  required={true}
```

Comment:
can we not use this prop, `required`? If required is true, then show the label?

Reply:
yes, refactor follow your advice, it is much clear now